portal/utils/format.ts

```diff
@@ -23,6 +23,42 @@ export const formatNumber = (value: number | string) =>
 export const formatFiatNumber = (value: number | string) =>
   fiatRounder(value, { shouldFormat: true })
 
+export const formatPastTime = function (
+  secondsFromNow: number,
+  locale: string,
+) {
+  // We need to iterate from the largest unit to the smallest unit - so I prefer
+  // so sort them like that
+  /* eslint-disable sort-keys */
+  const thresholdsInSeconds: Partial<
+    Record<Intl.RelativeTimeFormatUnit, number>
+  > = {
+    // Defined to loosely match blockscout relative timestamps - all converted to seconds
+    // See https://github.com/hemilabs/blockscout-frontend/blob/4f993ce5e62382ea98f42c6d79818630af331b9d/lib/date/dayjs.ts#L13
+    year: 28944000, // 11 months, assuming 365 days - 30 days
+    month: 2419200, // 4 weeks
+    week: 518400, // 4 days
+    day: 82800, // 23 hours
+    hour: 3540, // 59 minutes
+    minute: 59, // 59 seconds
+    second: 1, // 1 second
+  }
+  /* eslint-disable sort-keys */
```

Comment:
Should it not be `/* eslint-enable sort-keys */` here?
```suggestion
  /* eslint-enable sort-keys */
```

Reply:
you're correct! Good catch. Fixed here  [2a48184](https://github.com/hemilabs/ui-monorepo/pull/1169/commits/2a481848db67ebe97a21dc3f21bb6df9fa1e66c2)